lib/symbol-provider.coffee

```diff
@@ -1,12 +1,13 @@
 # This provider is currently experimental.
 
 _ = require 'underscore-plus'
-fuzzaldrin = require 'fuzzaldrin'
-fuzzaldrinPlus = require 'fuzzaldrin-plus'
-{CompositeDisposable}  = require 'atom'
+{TextBuffer, Range, CompositeDisposable}  = require 'atom'
 {Selector} = require 'selector-kit'
 SymbolStore = require './symbol-store'
 
+# TODO: remove this when `onDidStopChanging(changes)` ships on stable.
+bufferSupportsStopChanging = -> typeof TextBuffer::onDidChangeText is "function"
```

Comment:
I am testing for the presence of `onDidChangeText` here because I couldn't find any other efficient heuristic to determine whether or not https://github.com/atom/text-buffer/pull/126 shipped. This is temporary, though, so it might not be that important.